pkg/jobs/jobspb/jobs.proto

```diff
@@ -402,7 +402,13 @@ message RestoreDetails {
 
 
 message RestoreProgress {
+  message RestoreProgressFrontierEntry {
```

Comment:
could you add a comment that explains exactly what this tracks? I think it's all completed spans, correct?

Reply:
Yes!